test/framework/management/kind/mgmt.go

```diff
@@ -179,6 +179,12 @@ func getKubeconfigPath(ctx context.Context, name string) (string, error) {
 
 // ClientFromRestConfig returns a controller-runtime client from a RESTConfig.
 func (c *Cluster) ClientFromRestConfig(restConfig *rest.Config) (client.Client, error) {
+	fmt.Println("===============")
+	fmt.Println("CHECKING SCHEME AGAIN BEFORE CREATING CLIENT")
+	for k, v := range c.Scheme.AllKnownTypes() {
+		fmt.Printf("%q : %q\n", k, v)
+	}
+	fmt.Println("===============")
 	cl, err := client.New(restConfig, client.Options{Scheme: c.Scheme})
```

Comment:
You need to create `api.NewDynamicRESTMapper` and pass it in to the options as the `Mapper`

Reply:
That worked! Thanks so much for the tip! I was stuck on this problem for a few days now.